src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -1455,12 +1455,15 @@ void IslandWindow::_enterQuakeMode()
     const til::size ncSize{ GetTotalNonClientExclusiveSize(dpix) };
     const til::size availableSpace = desktopDimensions + ncSize;
 
+    // GH#10201 - The borders are still visible in quake mode, so make us 1px
+    // smaller on either side to account for that, so they don't hang onto
+    // adjacent monitors.
     const til::point origin{
-        ::base::ClampSub<long>(nearestMonitorInfo.rcWork.left, (ncSize.width() / 2)),
+        ::base::ClampSub<long>(nearestMonitorInfo.rcWork.left, (ncSize.width() / 2)) + 1,

```

Comment:
I really don't understand what this code is doing here. I'll approve it because it quite obviously can't break anything, but I fail to understand why we need to subtract half of ncSize. ncSize is the desktop size, minus the size of the window right? So why isn't this just set to `nearestMonitorInfo.rcWork.left` directly?

Reply:
ncSize is the size of the Window's invisible 8-pixel borders :smile: (except it's not always 8, and it isn't uniform)

Reply:
so we want to adjust based on the size of the invisible portion of the window border